esx_service/utils/auth_api.py

```diff
@@ -398,36 +410,57 @@ def _tenant_ls(name=None):
     error_info, tenant_list = get_tenant_list_from_db(name)
     return error_info, tenant_list
 
-def any_vm_already_exist(existing_vms, vms):
+def vm_already_exist(name, vms):
```

Comment:
I understand that. However, for the purpose of this method, we don't need the tenant name parameter. Actually for the error message I think we should log the tenant which contain this VM instead of the tenant to be added.